snowflake-tester/src/test/java/org/apache/sedona/snowflake/snowsql/TestFunctionsV2.java

```diff
@@ -973,6 +973,24 @@ public void test_ST_LengthSpheroid() {
         );
     }
 
+    @Test
+    public void test_ST_ForcePolygonCCW() {
+        registerUDFV2("ST_ForcePolygonCCW", String.class);
+        verifySqlSingleRes(
```

Comment:
@furqaankhan Your TestFunctionsV2 implementation is not following the correct format. Please pay close attention to the format of other functions in the same TestFunctionsV2 file.

Reply:
In short, you should use the constructor that has no `sedona` prefix. This kind of constructors are snowflake native constructors. See the explanation here: https://sedona.apache.org/1.5.1/tutorial/snowflake/sql/#interoperate-with-snowflake-native-functions